### PR TITLE
[Merged by Bors] - Added meaning of issue label prefixes

### LINKED ISSUE #7704: Add issue label letter meanings to contributing guide
## How can Bevy's documentation be improved?

The [contributing guide](https://github.com/bevyengine/bevy/blob/main/CONTRIBUTING.md) does not describe issue label letter meanings.

### PR DESCRIPTION
**Pull request by adam-shih**
# Objective

Fixes #7704 

## Solution

Added a small section that describes the meaning of each letter prefix for issue labels.

### CODE REVIEW

**Review by MrGVSV — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,6 +81,15 @@ Our merge strategy relies on the classification of PRs on two axes:
...
 * How complex is the implementation
 
+Each issue [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
+
+* A: Area
+* C: Category
+* D: Difficulty
+* O: Operating system
+* P: Priority
+* S: Status
```
Comment (harudagondi) [inline CONTRIBUTING.md:91]: Maybe add an explanation for each category? When I first contributed to Bevy I didn't understand what's the difference between A and C.

Comment (lewiszlw) [reply CONTRIBUTING.md:91]: Maybe add some example labels

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,6 +81,15 @@ Our merge strategy relies on the classification of PRs on two axes:
 * How controversial are the design decisions
 * How complex is the implementation
 
+Each issue [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
```
Comment (lewiszlw) [inline CONTRIBUTING.md:84]:
```suggestion
Each issue and pull request [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (alice-i-cecile) [reply CONTRIBUTING.md:91]: Example labels are a good way to do this.

**Review by alice-i-cecile — APPROVED** [review]
This LGTM, but the suggestions would be nice to grab :) I'll give you some time to tweak that.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mockersf) [reply CONTRIBUTING.md:84]: or just "each label"? it's what they're called in GitHub docs and UI

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Nice revisions :)

bors r+

**Comment by adam-shih** [comment]
Wasn't sure what might be the best/most used examples to add, so I kind of randomly chose.

**Review by harudagondi — APPROVED** [review]

**Comment by james7132** [comment]
bors retry

**Pull request closed without merging**